common/autoware_test_utils/config/test_behavior_velocity_planner.param.yaml

```diff
@@ -0,0 +1,11 @@
+/**:
```

Comment:
@yhisaki 
Please add comment about being able to use this file for other behavior velocity planner's module test